### PR TITLE
Schema updates: SpectralTypes and Gravities

### PR DESCRIPTION
**Pull request by dr-rodriguez**
This adds the SpectralTypes and Gravities tables, as described in #21 and #22. 
It also adds tests to verify that any entries to those tables do include those values and that at least one best spectral type has been flagged.
Note that spectral types are encoded as 10-character strings as opposed to numeric floats (ie, we should expect things like L3.5, not just 13.5).

In addition, there are minor updates to the README and to the generate_database script in case users want to use the later to build/refresh a postgres version of the SIMPLE database.

### CODE REVIEW

File: scripts/tutorials/generate_database.py [hunk 1]
```diff
@@ -7,14 +7,34 @@
...
+# SQLlite settings
+DB_TYPE = 'sqlite'
+DB_NAME = 'SIMPLE.db'
+
+# Set correct connection string
+if DB_TYPE == 'sqlite':
+    # First, remove the existing database in order to recreate it from the schema
+    # If the schema has not changed, this part can be skipped
+    if os.path.exists(DB_NAME):
+        os.remove(DB_NAME)
```
Comment (Will-Cooper) [inline scripts/tutorials/generate_database.py:24]: Is there a handle for if a user doesn't have write/ remove permissions?

File: simple/schema.py [hunk 2]
```diff
@@ -32,6 +33,25 @@ class Instruments(Base):
...
+    radio = 'radio'
+
+
+class Gravity(enum.Enum):
+    """Enumeration for gravity"""
+    a = 'alpha'
+    b = 'beta'
+    g = 'gamma'
+    bg = 'beta/gamma'
+    unknown = 'unknown'
```
Comment (Will-Cooper) [inline simple/schema.py:53]: What about the ones from NIR spectra? e.g. vl-g, int-g, fld-g.
There's also delta for optical

File: scripts/tutorials/generate_database.py (hunk 1 above)
Comment (dr-rodriguez) [reply scripts/tutorials/generate_database.py:24]: Nope, it's meant for you to run on your own machine or by the github action which already has permissions.

File: simple/schema.py (hunk 2 above)
Comment (dr-rodriguez) [reply simple/schema.py:53]: Yep, good points- I don't know all the possibilities so it's good to get feedback like this!

**Comment by dr-rodriguez** [comment]
I'll add the other gravity enumerations and change `best` to `adopted` in the table

File: tests/test_integrity.py [hunk 3]
```diff
@@ -259,6 +259,44 @@ def test_radialvelocities(db):
...
+    if len(t) > 0:
+        print("\nSpectral Type entries with incorrect 'best' labels")
+        print(t)
+    assert len(t) == 0
+
+
+def test_gravities(db):
+    # Tests against the Gravities table
+
+    # There should be no entries in the SpectralTypes table without a spectral type
```
Comment (dr-rodriguez) [inline tests/test_integrity.py:290]: Comment needs updating: Gravities

**Review by Will-Cooper — APPROVED** [review]

**Pull request merged**